controllers/admin/AdminImagesController.php

```diff
@@ -377,6 +436,17 @@ public function initModal(): void
         ];
     }
 
+    public function updateOptionPsImageFormat($value)
+    {
+        if ($this->access('edit') != '1') {
+            throw new PrestaShopException($this->trans('You do not have permission to edit this.', [], 'Admin.Notifications.Error'));
+        }
+
+        if (!$this->errors && $value) {
+            Configuration::updateValue('PS_IMAGE_FORMAT', implode(',', $value));
```

Comment:
Well, I see that you did override the field saving already, so the thing remaining is a different rendering of the input field in the HTML :)